src/adcm_client/objects.py

```diff
@@ -695,6 +756,17 @@ def service_delete(self, service: "Service"):
         with allure_step(f"Remove service {service.name} from cluster {self.name}"):
             self._subcall("service", "delete", service_id=service.id)
 
+    def get_unaccepted_service_licenses(self) -> List[License]:
+        return list(
+            map(
+                partial(License, api=self._api),
+                filter(
+                    lambda sp: sp.license == LicenseStatus.UNACCEPTED.value,
+                    self.bundle().service_prototype_list(),
+                ),
+            )
+        )
```

Comment:
Unreadable code. Why do we need the `get_unaccepted_service_licenses()` method?

Reply:
Reworked with list comprehension

We need this method to get unaccepted licenses of cluster